yoti-sdk-api/src/main/java/com/yoti/api/client/spi/remote/call/ErrorDetails.java

```diff
@@ -0,0 +1,94 @@
+package com.yoti.api.client.spi.remote.call;
+
+import java.util.Objects;
+
+import com.fasterxml.jackson.annotation.JsonProperty;
+
+public final class ErrorDetails {
+
+    @JsonProperty(Property.ERROR_CODE)
+    private String code;
+
+    @JsonProperty(Property.DESCRIPTION)
+    private String description;
```

Comment:
Isn't this class supposed to _de_-serialize a JSON? If so the `JsonProperties` should go in the builder methods (assuming the builder is intended to be used to de-serialize the JSON)

If this class is intended to serialize to JSON then nevermind (if both then the comment regarding the builder being used for de-serialization still applies).